app/src/androidTest/java/com/github/dedis/student20_pop/ui/OrganizerFragmentTest.java

```diff
@@ -59,6 +64,93 @@ public void canCancelAddEvent() {
+        onView(withId(R.id.organization_name_editText)).perform(clearText());
+        onView(withId(R.id.organization_name_editText)).perform(typeText(stringToBeTyped));
+        onView(withId(R.id.properties_edit_confirm)).perform(click());
+        onView(allOf(withText(stringToBeTyped), withId(R.id.organization_name))).check(matches(isDisplayed()));
+    }
+
+    @Test
+    public void canCancelEditProperties() {
+        onView(withId(R.id.tab_properties)).perform(click());
+        onView(withId(R.id.properties_view)).check(matches(isDisplayed()));
+        onView(withId(R.id.edit_button)).perform(click());
+        onView(withId(R.id.organization_name_editText)).perform(clearText());
+        onView(withId(R.id.properties_edit_cancel)).perform(click());
+        onView(withId(R.id.properties_view)).check(matches(isDisplayed()));
+    }
+
+    @Test
+    public void clickOnDeleteOpensDialog() {
+        onView(withId(R.id.tab_properties)).perform(click());
+        onView(withId(R.id.properties_view)).check(matches(isDisplayed()));
+        onView(withId(R.id.edit_button)).perform(click());
+        onData(anything()).inAdapterView(withId(R.id.witness_edit_list))
+                .atPosition(0).onChildView(withId(R.id.image_button_delete_witness)).check(matches(isDisplayed())).perform(click());
+        onView(withText("Delete ?")).check(matches(isDisplayed()));
+    }
+
+    @Test
+    public void canCancelDeleteWitness() {
+        onView(withId(R.id.tab_properties)).perform(click());
+        onView(withId(R.id.properties_view)).check(matches(isDisplayed()));
+        onView(withId(R.id.edit_button)).perform(click());
+        onData(anything()).inAdapterView(withId(R.id.witness_edit_list))
+                .atPosition(0).onChildView(withId(R.id.image_button_delete_witness)).check(matches(isDisplayed())).perform(click());
+        onView(withText("Delete ?")).check(matches(isDisplayed()));
+        onView(withText(R.string.button_cancel)).perform(click());
+        onData(anything()).inAdapterView(withId(R.id.witness_edit_list))
+                .atPosition(0).onChildView(withId(R.id.image_button_delete_witness)).check(matches(isDisplayed()));
+    }
+
+    @Test
+    public void canDeleteWitness() {
+        onView(withId(R.id.tab_properties)).perform(click());
+        onView(withId(R.id.properties_view)).check(matches(isDisplayed()));
+        onView(withId(R.id.edit_button)).perform(click());
+        onData(anything()).inAdapterView(withId(R.id.witness_edit_list))
+                .atPosition(0).onChildView(withId(R.id.image_button_delete_witness)).check(matches(isDisplayed())).perform(click());
+        onView(withText("Delete ?")).check(matches(isDisplayed()));
+        onView(withText(R.string.button_confirm)).perform(click());
+    }
+
+    @Test(expected = IllegalArgumentException.class)
+    @Ignore("TODO : Is it right to allow empty string for a lao's name ?")
+    public void confirmNullLaoTitleThrowsException() {
```

Comment:
A LAO with an empty name is not a problem in our implementation, maybe we can change that on another issue

Reply:
It would be weird a LAO with an empty name, no ?

Reply:
Of course, I agree with you. But this should be modified and tested in another issue

Reply:
Okay I was not sure with your first comment. I'll remove this test and open an issue